daemon/logger/jsonfilelog/jsonfilelog.go

```diff
@@ -56,7 +56,7 @@ func init() {
 // New creates new JSONFileLogger which writes to filename passed in
 // on given context.
 func New(ctx logger.Context) (logger.Logger, error) {
-	log, err := os.OpenFile(ctx.LogPath, os.O_RDWR|os.O_APPEND|os.O_CREATE, 0600)
+	log, err := os.OpenFile(ctx.LogPath, os.O_RDWR|os.O_APPEND|os.O_CREATE, 0666)
```

Comment:
If we're going to adjust permissions, I think `0640` would a better value. 666 is too open. By default it's not an issue as the directory these logs get written to is `0700` & owned by `docker`. But if the daemon gets configured to write outside that directory, these logs become writable to the world.